packages/web-components/src/dialog/README.md

```diff
@@ -10,11 +10,9 @@
 
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29648&commitId=9a37cd8c5bbe06a50653d0f2eecfe64342ae8de2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Switch</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Switch.Checked - RTL.normal.chromium.png|97| Changed|
|Switch.Checked - RTL.hover.chromium.png|104| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>